apiserver/plane/api/serializers/issue.py

```
@@ -43,7 +43,7 @@ def is_uuid(value):
 class IssueCustomPropertySerializer(BaseSerializer):
     class Meta:
         model = IssueCustomProperty
-        fields = ["key", "value", "issue_type_custom_property", "id"]
+        fields = ["key", "value", "data_type", "int_value", "bool_value", "date_value", "issue_type_custom_property", "id"]
```

Pylint (3.3.7)

[convention] Line too long (123/100)

(C0301)